File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = [-1 for n in range(0,2*len(nums))]
        for n in range(0,len(nums)):
            ans[n] = nums[n]
            ans[n+len(nums)] = nums[n]

        return ans

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    topKFrequent(nums, k) {
        const frequencyMap = {};

        for (let num of nums){
            if(!frequencyMap[num]){
                frequencyMap[num] = 1;
                continue;
            }
            frequencyMap[num] += 1;
        }

        const freq = Object.entries(frequencyMap)
        .sort((a, b) => b[1] - a[1])
        .slice(0, k)
        .map(([key]) => key);
        console.log(freq)
        return freq;
    }
}
